frontend/pages/settings/personal-access-tokens/new.tsx

```diff
@@ -21,16 +24,48 @@ import Title from 'src/common/Title'
 import Link from 'src/Link'
 import MessageAlert from 'src/MessageAlert'
 import TokenDialog from 'src/settings/authentication/TokenDialog'
-import { TokenActions, TokenActionsKeys, TokenInterface, TokenScope } from 'types/types'
+import { TokenAction, TokenActionKeys, TokenInterface, TokenScope } from 'types/types'
 import { getErrorMessage } from 'utils/fetcher'
 import { plural } from 'utils/stringUtils'
 
+const [TokenReadAction, TokenWriteAction] = Object.values(TokenAction).reduce<Record<string, TokenActionKeys>[]>(
+  ([readActions, writeActions], action) => {
+    let groupedActions = [readActions, writeActions]
+    const [name, kind] = action.split(':')
+
+    if (kind === 'read') {
```

Comment:
Is 'read' and 'write' worth typing? I'm on the fence because they're so commonly used that hard-coding might be acceptable.